site/src/utils/router.js

```diff
@@ -160,7 +160,15 @@ class Router {
    * @param {FetchEvent} event
    */
   listen(event) {
-    event.respondWith(this.route(event))
+    // when testing just respond to the fake testing domain
+    if (process.env.PW_TEST) {
+      const url = new URL(event.request.url)
+      if (url.hostname === 'testing.com') {
```

Comment:
@hugomrdias we should probably use `testing.nft.storage`

```suggestion
      if (url.hostname === 'testing.nft.storage') {
```